images/linux/scripts/installers/pypy.sh

```diff
@@ -52,6 +52,11 @@ function InstallPyPy
 
     echo "Create additional symlinks (Required for UsePythonVersion Azure DevOps task)"
     cd $PYPY_TOOLCACHE_VERSION_ARCH_PATH/bin
+
+    PYPY_FULL_VERSION=$(./$PYPY_MAJOR -c "import sys;print('{}.{}.{}'.format(*sys.pypy_version_info[0:3]))")
```

Comment:
I think it should be moved above where you get PYTHON_FULL_VERSION

Reply:
Done.